### DISCUSSION TITLE
Cannot Enable Spotify Renderer

### DISCUSSION THREAD

**Discussion by digixmax** [rern/rAudio#698] [Fixed Issues] [answered]
I am not able to activate the Spotify renderer on rAudio-1-64bit-20211203: after successfully configuring the App on Spotify Developers portal then entering the App ID and Client Secret, the renderer "on/off" slider would not stay "on" and instead revert to "off" (the status of the spotifyd is showed as "inactive/dead").

Any info/pointer on what I might have missed would be greatly appreciated.

Also, why is the renderer activation/configuration changed in this release from what it was in earlier releases (e.g., I don't recall having to configure App ID and Client Secret in rAudio-1-RPi4-20210813)?

TIA.

**Comment by digixmax** [comment]
FWIW - my current workaround while waiting for a proper solution is to restore the backup configuration I used with the previous release rAudio-1-RPi4-20210813.

**Comment by Raitsa** [comment]
Somewhat related, never seen this happen before and I only ever use Firefox:

rAudio 1 • 20211210
5.10.82-4 • aarch64
20211201-1
Raspberry Pi 4 Model B Rev 1.4
BCM2711 • 3.7GiB

https://i.imgur.com/OZvHgz1.png

**Comment by digixmax** [comment, marked as answer]
Issue was solved with update to v211210 per rern's instruction https://github.com/rern/rAudio-1/discussions/628#discussioncomment-1802816.

**Comment by Raitsa** [comment]
I just solved my issue by using an Ipad + Safari to paste Client ID and Client Secret to enable Spotify in my new rAudio install.
